src/styles/base/_root.scss

```diff
@@ -2,15 +2,16 @@
 
 :root {
 	font-size: 62.5%;
+	overflow: hidden;
 }
 
-
 body {
 	font-family: $font-family-primary;
 	font-size: $font-size-base;
 	line-height: $line-height-lg;
 	margin: 0;
 	overflow-y: scroll;
+	max-height: 100vh;
 }
```

Comment:
Deze aanpassingen her-introduceren de "jumpy" page width (geen permanente scrollbalk).
`develop`
![image](https://user-images.githubusercontent.com/90180417/166663151-9643a5f0-15b4-45ca-bd1d-8b4fb1f0537e.png)
`bugfix/ARC-162-fix-zendesk-widget-position`
![image](https://user-images.githubusercontent.com/90180417/166663263-aa5bbefd-4e85-41c6-ba77-847c1b74ffcc.png)

Reply:
Dat lijkt dan enkel een issue op mac
op windows is er wel altijd een scrollbalk.

![image](https://user-images.githubusercontent.com/1710840/166665845-1987d927-e85f-45f9-9d46-db0adb54aab6.png)

![image](https://user-images.githubusercontent.com/1710840/166665851-cd1658c4-7762-4e2d-9973-a83daf3bcb98.png)